Article/Article.js

```diff
@@ -1,20 +1,21 @@
 // Because classes are not hoisted you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
 
 class Article {
-  constructor(domElement) {
+  constructor(art) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.art = art;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = art.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.innerText = 'expand';
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', (e) => { this.expandArticle(e) })
```

Comment:
since the expandArticle method is not using or even looking for an parameter, there's no need to pass the click event 'e' into this function call.

```
this.expandButton.addEventListener('click', () => this.expandArticle() );
```